cmd/genji/main.go

```diff
@@ -26,38 +25,71 @@ func main() {
 		},
 	}
 
+	app.Commands = []*cli.Command{
+		{
+			Name:      "insert",
+			Usage:     "Allows inserting documents from the command line",
+			UsageText: "genji insert [--file, -f] [db path] [--table, -t] [table name] [arguments...]",
+			HideHelp:  true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:      "file",
```

Comment:
I realize I made a mistake in the spec, sorry about that.
The way we deal with the database should be the following:

- `--db`: Path of the database
- `--engine, -e`: Name of the engine

Reply:
What I understand is  what you mean is to replace `--file, -f` by `--db` and keep it required and with no alias.
`--engine, -e` is not required. if `-e` is not specified `bolt` is used a default engine.

Reply:
I think it will be better to remove the `--bolt` and `badger` options and replace them by `-e` instead. But not necessarily in this PR

Reply:
Got it. 
So replace `--bolt` and `--badger` by `-e` in a new PR.